recipes/spirv-headers/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(test_package)
+
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Can be removed since included headers and your cpp file don't use C++11.

Reply:
Done!